views/js/runner/plugins/security/fullScreen.js

```diff
@@ -23,7 +23,8 @@ define([
     'lodash',
     'i18n',
     'taoTests/runner/plugin',
-    'taoQtiTest/runner/config/states'
+    'taoQtiTest/runner/config/states',
+    'css!taoTestRunnerPluginsCss/fullscreen'
```

Comment:
Where did you register `taoTestRunnerPluginsCss` alias? How the system know what is the path?

Reply:
Nowhere, it is generated by grunt task and used [same way](https://github.com/oat-sa/extension-tao-nfer/blob/fc6d1dcbae6d558f38beb1ffc0304e4af076b003/views/js/qtiCreator/plugins/panel/itemTemplates.js#L32) as in another extensions

Reply:
I'm 100% sure there is no connection with grunt task. But I do not know, where but it is automatically added to `/config/tao/client_lib_registry.conf.php`. I think Tao has an automatic solution for it.